components/cards/card.html

```diff
@@ -0,0 +1,292 @@
+                style="width: 100%; height: 562px; border:0; transform: scale(1); overflow:hidden;"
+                sandbox="allow-scripts allow-same-origin">
+            </iframe>
+            <hr class="hr-margin">
+            <h2>Cards with text overlay</h2>
+            <p class="heading-details">
+                In this card we have Out of Stock text over image of card.
+            </p>
+            <div class="card-container">
+                <div class="card">
+                    <div class="badge-section"></div>
+                    <span class="badge-best-seller display-none">Best seller</span>
+                    <span class="display-none">
+                        <i class="far fa-times-circle cross-icon"></i>
+                    </span>
+                    <span class="out-of-stock">
+                        Out Of Stock
+                    </span>
+                    <div class="card-main-section">
+                        <img src="/images/IMG_20220205_233718.png" class="product-img" alt="">
+                        <main class="middle">
+                            <p class="para">Apple</p>
+                            <div class="small-1">iPhone 13 (256GB) - Midnight</div>
+                            <div class="small-2">₹78900
+                                <span class="real-price">MRP: ₹79900(1% OFF)</span>
+                            </div>
```

Comment:
if there is only 1 element inside div, span, then i guess div container is not needed maybe